sqlalchemy_bigquery/_helpers.py

```diff
@@ -54,6 +55,9 @@ def create_bigquery_client(
         )
         credentials = credentials.with_scopes(SCOPES)
         default_project = credentials.project_id
+    elif credentials_access_token:
+        credentials = oauth_credentials.Credentials(credentials_access_token)
+        _, default_project = google.auth.default(scopes=SCOPES)
```

Comment:
This is potentially a slow operation. I we might consider only doing this if `project_id` is `None`. Or maybe add a `if default_project is None` to the `if project_id is None` if statement below?